src/views/GrampsjsViewMap.js

```diff
@@ -154,6 +163,11 @@ export class GrampsjsViewMap extends GrampsjsView {
     }
   }
 
+  _handleTimeSliderChange(event) {
+    this._dateRangeStart = event.detail.start
+    this._dateRangeEnd = event.detail.end
```

Comment:
It looks like this code would have access to the Leaflet map via `map._map`. From there, it’s another hop to get to the MapLibre `Map` object, which can set filters and such.

Here’s some public domain code for filtering an OHM style by date tags:

https://github.com/OpenHistoricalMap/ohm-embed-1yr/blob/d8d7f2362bfd87521c4559cf59885a1282e52621/embed-map-single-year.js#L77-L170

At some point, we should package it up as a proper library, independent of any time slider UI, but for now it’s stable and straightforward enough to copy-paste around.